client/blocks/design-sidebar/index.jsx

```diff
@@ -0,0 +1,36 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import RootChild from 'components/root-child';
+
+class DesignSidebar extends Component {
+	constructor( props ) {
+		super( props );
+	}
```

Comment:
is this needed if we don't do anything with it? I was thinking that without a constructor, the call was implicit. maybe not.